modules/ai-agents/pages/mcp-server/quickstart.adoc

```diff
@@ -153,7 +172,17 @@ Only tools with the specified `--tag` are exposed. This helps you:
 
 == Connect Claude Code to your MCP server
 
-To connect Claude Code to your MCP server, you need to expose a live event stream that Claude can consume. This is done using the link:https://www.npmjs.com/package/mcp-remote[`mcp-remote` utility^], which bridges your local service to Claude's MCP interface. `mcp-remote` is a lightweight bridge that turns any streaming HTTP endpoint into a source of MCP-compatible messages.
+Now that your MCP server is running and exposing tools, you can connect Claude Code so it can discover and use them.
+
+Claude Code communicates with MCP servers using the standard input/output (stdio) transport protocol. Because your Redpanda Connect MCP server exposes tools over HTTP with Server-Sent Events (SSE), you need a bridge to convert between these protocols. The link:https://www.npmjs.com/package/mcp-remote[`mcp-remote` utility^] provides this bridge.
+
+When you connect Claude Code:
+
+- `mcp-remote` subscribes to your server's SSE endpoint (`http://localhost:4195/sse`).
+- It translates HTTP/SSE messages into stdio format that Claude understands.
+- Claude automatically discovers your tools and their metadata.
+- You can ask Claude in natural language to use your tools.
+- Claude calls the appropriate tool based on your request.
 
```

Comment:
@JakeSCahill this looks weird, like the steps are related to the last item in the list above. Suggest adding an intro sentence after the unordered list, and then the ordered list will be left-aligned (instead of nested under the unordered list)